source/include/jobs.h

```diff
@@ -19,7 +19,6 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-
 /**
  * @file jobs.h
  * @brief Integration for the AWS IoT jobs APIs
```

Comment:
The word "_Integration_" can be unclear in the API header file. 
Can we use something like "Client library APIs for the AWS IoT Jobs service."

Reply:
changed